### DISCUSSION TITLE
Monthly Update - June 2024

### DISCUSSION THREAD

**Discussion by boundless-forest** [ringdao-community-guild/discussion#162] [Show and tell]
# Monthly Update - June 2024

## Protocol Updates

- Deprecate the BLS12381 precompile. https://github.com/darwinia-network/darwinia/pull/1511

## Darwinia Ecosystem Applications and Infrastructure Updates

- The Darwinia assethub bridge app is now upgraded and rebranded as Darwinia Paralink.https://x.com/DarwiniaNetwork/status/1803973713953792497
- $RING cross-chain transfers between Darwinia and Hydration are now supported! https://x.com/DarwiniaNetwork/status/1803693628100845714
- Community RPC URL change. https://github.com/orgs/dcdao/discussions/151
- Darwinia partner [Envio](https://envio.dev/) supports devs and data analysts building on the Darwinia with cutting-edge data indexing infrastructure and tools. https://x.com/envio_indexer/status/1802620396069331146
- Crab chain’s new explorer is ready for use. https://x.com/DarwiniaNetwork/status/1802634860625829916
- 400 Million [$RING](https://x.com/search?q=%24RING&src=cashtag_click) Burn! https://x.com/DarwiniaNetwork/status/1800371299921105368
- RING is tradeable on Hydration. https://x.com/hydration_net/status/1801358004710756839
- The new testnet, Darwinia Koi, will replace Pangolin as the new testnet. https://x.com/DarwiniaNetwork/status/1800823175548436661
- Darwinia is one of the exhibitors ahead of Polkadot Decoded 2024 showcasing cutting-edge cross-chain technology.
- Darwinia governance platform on Subsquare has changed. https://x.com/DarwiniaNetwork/status/1798275056050856265

**Comment by Laki389** [comment]
Hi, I think this report should be published on Darwinia Medium as well, as before. :+1:

@wuminzhe @boundless-forest

**Reply by boundless-forest** [reply]
https://medium.com/darwinianetwork/darwinia-monthly-recap-june-2024-88597c9d281b Done
